package.json

```diff
@@ -11,35 +11,36 @@
     "url": "https://github.com/thisdot/open-source/issues"
   },
   "scripts": {
-    "ng": "nx",
-    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2020 browser module main",
-    "start": "nx serve showcase",
-    "build": "nx build",
-    "test": "nx test",
-    "lint": "nx workspace-lint && nx lint",
-    "lint:fix": "nx workspace-lint && nx lint --fix",
-    "e2e:chrome": "nx e2e showcase-e2e --browser=chrome",
-    "e2e:edge": "nx e2e showcase-e2e --browser=edge",
-    "e2e:ff": "nx e2e showcase-e2e --browser=firefox",
-    "e2e:debug": "nx e2e showcase-e2e --watch",
+    "affected": "nx affected",
```

Comment:
why did you reorder all the scripts? it's hard to track what has changed

Reply:
`npm audit` rearranged them alphabetically

Reply:
do you suggest i revert them back to how it previously was? @ktrz

Reply:
reverted